extensions/voice/org.eclipse.smarthome.voice.mactts/src/main/java/org/eclipse/smarthome/voice/mactts/internal/MacTTSService.java

```diff
@@ -89,17 +90,21 @@ public AudioStream synthesize(String text, Voice voice, AudioFormat requestedFor
      */
     private final Set<Voice> initVoices() {
         Set<Voice> voices = new HashSet<Voice>();
+        InputStreamReader inputStreamReader = null;
+        BufferedReader bufferedReader = null;
         try {
             Process process = Runtime.getRuntime().exec("say -v ?");
-            InputStreamReader inputStreamReader = new InputStreamReader(process.getInputStream());
-            BufferedReader bufferedReader = new BufferedReader(inputStreamReader);
+            inputStreamReader = new InputStreamReader(process.getInputStream());

```

Comment:
Because in this case I found it easier to do it with a finally block. The ` Process process = Runtime.getRuntime().exec("say -v ?");` line should be surrounded with  try-catch but it is not a closeable object, so it cannot be put in the try-with-recources. So if I was not using the finally block, I would have to put one try-catch block in addition to the try-with-recources block.